cmd/relationtuple/output.go

```diff
@@ -0,0 +1,140 @@
+package relationtuple
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ory/keto/ketoapi"
+	rts "github.com/ory/keto/proto/ory/keto/relation_tuples/v1alpha2"
+)
+
+type (
+	Collection struct {
+		protoRelations []*rts.RelationTuple
+		apiRelations   []*ketoapi.RelationTuple
```

Comment:
I find this type that has either proto or api relations hard to reason about. Can both be nil? Can both be non-nil? Unfortunately Go does not give us an "either".

Instead, how about we just store the `apiRelations` here. Then we can in the `NewProtoCollection` constructor call `Normalize` once.